src/platform/unittest/TlsTest.cpp

```diff
@@ -777,6 +779,7 @@ TEST_F(TlsTest, Handshake)
 #ifndef QUIC_DISABLE_PFX_TESTS
 TEST_F(TlsTest, HandshakeCertFromFile)
 {
+    ASSERT_NE(nullptr, CertParamsFromFile);
```

Comment:
`SetUpTestSuite()` will still fail, causing all tests to be skipped. Perhaps we should update that to only generate `CertParamsFromFile` if `PfxPath` is present, and then keep this line here, so only this test fails?

Reply:
Probably a good idea. Fixed.